### PR TITLE
Add instructions for deploying the OpenTelemetry Operator.

### PR DESCRIPTION
**Pull request by dinooliva**
### What does this PR do?

Adds instructions for deploying the OpenTelemetry Operator.

### Motivation

Our customers are seeking guidance on using the OpenTelemetry Operator in combination with OTel Collector configured to export to Datadog.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md [hunk 1]
```diff
@@ -285,6 +285,17 @@ For Gateway deployments:
...
 
+### OpenTelemetry Operator for Kubernetes
+
+To use the OpenTelemetry Operator:
+1. See the [official documentation for deploying the OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator#readme). As detailed there, you need to deploy the certificate manager in addition to the Operator.
+
+2. Deploy the OpenTelemetry Collector using one of the standard configurations:
+   * [Daemonset Deployment](#daemonset-deployment)
+   * [Gateway Deployment](#gateway-collector-service)
+
```
Comment (dineshg13) [inline content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md:296]: May be adding  a deploy config with operator will make it easier for users/

File: content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md [hunk 2]
```diff
@@ -285,6 +285,17 @@ For Gateway deployments:
...
+**Note:** you are **strongly encouraged** to use the daemonset deployment with the OpenTelemetry Operator otherwise host metrics may not work.
```
[the remark this answers is not in the document]
Comment (dinooliva) [reply content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md:297]: I've just add Prashant as a reviewer.

**Review by pj-datadog — APPROVED** [review]

File: content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md (hunk 1 above)
Comment (dinooliva) [reply content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md]: Done.

**Comment by alai97** [comment]
Created https://datadoghq.atlassian.net/browse/DOCS-4135!

**Review by kayayarai — COMMENTED** [review]
Hi @dinooliva ! I made a couple wording and formatting corrections, and moved the note about host metrics up to a point where the reader is actually making the decision. Let me know if this looks okay to you, and we can merge it when you're ready

**Pull request merged**